src/sysupdate/sysupdate-resource.c

```diff
@@ -160,6 +173,29 @@ static int resource_load_from_directory(
         return 0;
 }
 
+static int resource_load_from_directory(
+                Resource *rr,
+                mode_t m) {
+        _cleanup_closedir_ DIR *d = NULL;
+
+        assert(rr);
+        assert(IN_SET(rr->type, RESOURCE_TAR, RESOURCE_REGULAR_FILE, RESOURCE_DIRECTORY, RESOURCE_SUBVOLUME));
+        assert(IN_SET(m, S_IFREG, S_IFDIR));
+
+        d = opendir(rr->path);
+
+        if (!d) {
+                if (errno == ENOENT) {
+                        log_debug("Directory %s does not exist, not loading any resources.", rr->path);
```

Comment:
actually, we should use `log_debug_errno(errno, ` here too. When we generate structured logging the error will be passed in the ERRNO= field, hence as a matter of coding tyle we'll always use the `log_xyz_errno()` flavour of the log calls if we have an error number.

(i understand this is just moving around of pre-existing code, but if you touch it, please also update to newest coding style)